composer.json

```diff
@@ -21,41 +21,41 @@
     ],
     "homepage": "https://github.com/sonata-project/sonata-doctrine-extensions",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "doctrine/dbal": "^2.13 || ^3.1",
-        "doctrine/persistence": "^1.3.6 || ^2.0 || ^3.0"
+        "php": "^8.0",
+        "doctrine/dbal": "^3.0",
+        "doctrine/persistence": "^2.5 || ^3.0"
     },
     "require-dev": {
         "doctrine/common": "^2.7 || ^3.0",
-        "doctrine/doctrine-bundle": "^2.0",
-        "doctrine/mongodb-odm": "^2.0",
-        "doctrine/orm": "^2.5",
+        "doctrine/doctrine-bundle": "^2.6",
+        "doctrine/mongodb-odm": "^2.2",
+        "doctrine/orm": "^2.10",
         "friendsofphp/php-cs-fixer": "^3.4",
-        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.7.6",
```

Comment:
is this dependency still needed? I think it was used along with phpcr

Reply:
On it https://github.com/sonata-project/sonata-doctrine-extensions/pull/417